### DISCUSSION TITLE
Question regarding melspectrogram settings for  Models and universal Vocoders

### DISCUSSION THREAD

**Discussion by michaellin99999** [coqui-ai/TTS#1030] [General Q&A]
do the Freq_min and Freq_max audio parameters  for the TTS model need to match that of the pertained universal vocoders? 
if the model audio parameters we set to Freq_min 200 to Freq_MAX 3200,    will this still work with universal vocoder in the repo? or would we need to train our own?

**Comment by erogol** [comment]
you need to train a new vocoder matching the audio parameters.

**Reply by michaellin99999** [reply]
do we use the dataset we trained the TTS model to train the vocoder? or should it be the output of the TTS model used to train the vocoder?

Would training the Vocoder using LJspeech but tuned to the audio parameters above also work?

**Reply by erogol** [reply]
> do we use the dataset we trained the TTS model to train the vocoder? or should it be the output of the TTS model used to train the vocoder?

Yes using specs from the encoder model yields slightly better results.

> Would training the Vocoder using LJspeech but tuned to the audio parameters above also work?

Yes, you can fine-tune the model with the new parameters.

**Reply by michaellin99999** [reply]
how many steps/epochs will training vocoder usually take? right now after doing 18000 steps using a rtx 3090 we still are getting static noise. we made sure the Freq_min 200 to Freq_MAX 3200 are the same and also our dataset is extra clean.   vocoder we trying to train is hi-fi-gan
